physics/module_mp_thompson_make_number_concentrations.F90

```diff
@@ -35,7 +36,7 @@ elemental real function make_IceNumber (Q_ice, temp)
       !IMPLICIT NONE
       REAL, PARAMETER:: Ice_density = 890.0
       !REAL, PARAMETER:: PI = 3.1415926536
-      real, intent(in):: Q_ice, temp
+      real(kind_phys), intent(in):: Q_ice, temp
```

Comment:
Why introduce using mixed precision in this module?

Reply:
@joeolson42 feel free to roll this back. I talked with Dustin and will add this in a new PR.

Reply:
@joeolson42 @AndersJensen-NOAA Would you like me to revert this change for final testing of this PR?